charts/zwavejs2mqtt/0.0.1/questions.yaml

```diff
@@ -1,14 +1,16 @@
 groups:
   - name: "Container Image"
-    description: "configure container image"
+    description: "Image configuration"
   - name: "Configuration"
-    description: "additional container configuration"
+    description: "Container configuration"
   - name: "Networking"
-    description: "Networking / service configuration"
-  - name: "Storage"
-    description: "configure app volume mounts"
```

Comment:
It has been (re)named storage, project wide.

Reply:
I thought as long the groups defined at the top match the groups used in the variable definitions, these groups could be determined per app.

I changed this group in zwavejs2mqtt to Volume Mounts because is this section includes devices, which are not storage.

I thought in this case, I thought volume mount was a more appropriate title for the group. -- I have confirmed using multiple group names that I have made up, these names can be arbitrarily unless you have added something to the test that will intentionally make it fail.

Reply:
Basically, these group name should only be related to how the questions is rendered. 

I might be mistaken, but as far as I can tell these group name have zero effect on the variables defined within them and the resulting app. They are strictly visual

Reply:
Its my personal preference as a maintainer, not a technical requirement.

I need to be able to bulk edit them later.

Reply:
I understand your point. Further more, I like like the idea of having some standard questions whenever possible as this should also help others easily contribute basic apps

The problem with grouping here is stated above, however I'm still not 100% satisfied trying to use Volume Mounts to group with devices either.

I don't want to make this WET again but what I duplicated the appVolumes to make an appDevices -- This would be redundant in the charts but would simply allow us to break the devices into a separate group on questions.